docs/Guides/Serverless.md

```diff
@@ -99,13 +107,131 @@ signature to be used as a lambda `handler` function. This way all the incoming
 events (API Gateway requests) are passed to the `proxy` function of
 [@fastify/aws-lambda](https://github.com/fastify/aws-lambda-fastify).
 
-### Example
+#### Example
 
 An example deployable with
 [claudia.js](https://claudiajs.com/tutorials/serverless-express.html) can be
 found
 [here](https://github.com/claudiajs/example-projects/tree/master/fastify-app-lambda).
 
+### Using @h4ad/serverless-adapter
```

Comment:
this pages is reserved for official modules, so I would prefer if we did not include any tutorials for technologies we do not maintain as part of this org. I'd be ok in adding a link to this other project as an alternative approach.

Reply:
Instead of putting the whole tutorial, should I put something like this?

--- 

#### Alternative Approach

If you need to integrate with more AWS services, take a look at [@h4ad/serverless-adapter](https://viniciusl.com.br/serverless-adapter/docs/main/frameworks/fastify) on Fastify to find out how to integrate.

---

Reply:
That would be great. The content of the tutorial could be on your website.